lib/stat_tracker.rb

```diff
@@ -54,6 +54,17 @@ def percentage_visitor_wins
     percentage.round(2)
   end
 
+  def find_tied_games
+    games.find_all do |game|
+      game.away_goals == game.home_goals
+    end
+  end
+
+  def percentage_ties
+    percentage = find_tied_games.count.fdiv(games.count) * 100
```

Comment:
@gabichuelas so based on your note in slack, it sounds like this should just be `percentage = find_tied_games.count.fdiv(games.count)` without the `* 100`, yes?

Reply:
@rrabinovitch yeah, and then we have to changed related tests to account for the change.